src/base/unicodestrings.h

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
You should revert changes to this file, read the git history and PR.

Reply:
I ended up reverting the change to this file, in the interest of moving this along, since it's clear there will be no more investigation into this. The file is now unchanged, with UTF-8 BOM.